headless/src/jsMain/kotlin/dev/fritz2/headless/components/modal.kt

```diff
@@ -27,7 +30,7 @@ class Modal : OpenClose(), WithJob {
     fun init() {
         opened.filter { it }.handledBy {
             PortalRenderContext.run {
-                portal(zIndex = PORTALLING_MODAL_ZINDEX, tag = RenderContext::dialog) { close ->
+                portal("display: contents", id = componentId, tag = RenderContext::dialog) { close ->
```

Comment:
It is generally not recommended to specify styling directly in an elements `class` attribute.
The above code results in an attribute like this: `class="display: contents"`.

Instead, the `style` attribute should be used:
```kotlin
attr("style", "display: contents;")
```